boards/st/stm32l562e_dk/doc/index.rst

```diff
@@ -273,6 +276,17 @@ Serial Port
 STM32L562E-DK Discovery board has 6 U(S)ARTs. The Zephyr console output is
 assigned to USART1. Default settings are 115200 8N1.
 
+TFT LCD screen and touch panel
+------------------------------
+
+The TFT LCD screen and touch panel are supported for the STM32L562E-DK Discovery board.
+They can be tested using :zephyr:code-sample:`lvgl` sample:
+
+.. zephyr-app-commands::
+
```

Comment:
sorry that sphinx is being uncooperative but adding that blank line is causing the directive to be interpreted different (I know this came from a good intention of trying to improve readbility :) )

Reply:
Good to know !  I don't know sphinx very well